File: S03-conceptos-basicos/CL13-CicloVidaFragment/app/src/main/java/com/udemyandroid/ciclovidafragment/MainActivity.java
package com.udemyandroid.ciclovidafragment;

import android.net.Uri;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity implements BlankFragment.OnFragmentInteractionListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    @Override
    public void onFragmentInteraction(Uri uri) {

    }
}

File: S13-kotlin/CL07-DadoAlAzar/app/src/main/java/com/androidavanzado/cl06_eventoclick/MainActivity.kt
package com.androidavanzado.cl06_eventoclick

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*
import java.util.*

class MainActivity : AppCompatActivity() {
    lateinit var dado: ImageView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnDado: Button = findViewById(R.id.buttonTirarDado)
        dado = findViewById(R.id.imageViewDado)

        btnDado.setOnClickListener {
            generarDadoAleatorio()
        }

    }

    private fun generarDadoAleatorio() {
        val randomInt = Random().nextInt(6) + 1

        val imageToLoad = when(randomInt) {
            1 -> R.drawable.dado_1
            2 -> R.drawable.dado_2
            3 -> R.drawable.dado_3
            4 -> R.drawable.dado_4
            5 -> R.drawable.dado_5
            else -> R.drawable.dado_6
        }

        dado.setImageResource(imageToLoad)
    }
}
